---
title: "Global Sewage Project"
author: "Antti Karkman"
---

```{r, echo=FALSE, include=FALSE}
knitr::opts_chunk$set(echo = TRUE, warning = FALSE, message = FALSE)
```

# Short Introduction 
The main objective of this project is to use untreated sewage as a marker for antibiotic resistance prevalence in clinical pathogens.  
We want to test whether sewage meatgenomic could be used to predict clinical resistance. 

## The Data
Metagenomic sequencing data from untreated sewage comes from an article in *Nature Communications* [Global monitoring of antimicrobial resistance based on metagenomics analyses of urban sewage.](https://doi.org/10.1038/s41467-019-08853-3)  
The raw sequence data from this study was downloaded from [ENA](https://www.ebi.ac.uk/ena) under project [ERP015409](https://www.ebi.ac.uk/ena/data/view/PRJEB13831).

The data consists of 234 sewage metagenomemes from 62 countries. The antibiotic resistance genes were annotated against [ResFinder v.3.1.0](https://bitbucket.org/genomicepidemiology/resfinder) database and *intI1* integrase gene against [MobileGeneticElementDatabase](https://github.com/KatariinaParnanen/MobileGeneticElementDatabase).  

The clinical resistance data for *E. coli* against four different classes of antibiotics, aminopenicillins, fluoroquinolones, 3rd generation cephalosporins and aminoglycosides was collected from different surveillance networks. Only data from more than 100 isolates was considered reliable and included in the actual analyses.  
In addition, aggregated resistance index from  [Collignon *et al.*, 2018](https://doi.org/10.1016/S2542-5196(18)30186-4) was used in the study.

Socio-economical factors were retrieved from [The World Bank Databank](https://databank.worldbank.org/source/health-nutrition-and-population-statistics).

All ARG and *intI1* integrase gene counts were normalized to the amount of sequence data (million bases). Since the clinical resistance data is on country level, mean of the normalized gene count data was taken for countries with more than one sewage sample. 

All data has been gathered to one data frame and is available in the `Data` folder. 

# Data Analyses 
Load all needed libraries. 
```{r, message=FALSE, warning=FALSE}
library(vegan)
library(tidyverse)
library(MASS)
#library(gridExtra)
library(GGally)
library(betareg)
library(lmtest)
library(pheatmap)
library(patchwork)
```

Read in the data and have a look. 
```{r}
# Data
load("../Data/GlobalSewage.R", verbose=TRUE)
glimpse(ResData)
```

## Main Article

### _E. coli_ clinical resistance models based on ten most common ARGs in _E. coli_ 

The first and maybe most obvious approach was to use the 10 most common ResFinder ARGs in publicly available _E. coli_ genomes in predicting clinical resistance in _E. coli_.  
Results from beta regression models for the four resistance categories and Figure 1 from the article presented below. 

**Clinical aminopenicillin resistance**
```{r}
tmp <- ResData %>% filter(`AP_(n)`>100)
tmp$y = tmp$`AP_(R%)`/100
breg <- betareg(y ~ mean_topColi, data=tmp, link="loglog")
summary(breg)


new_int <- data.frame(mean_topColi=seq(min(tmp$mean_topColi), max(tmp$mean_topColi)+0.05, 0.05))
new_int$y <- predict(breg, newdata=new_int)

p1 <- ggplot(tmp, aes(mean_topColi, y)) + geom_point(pch=21, size=5, fill="skyblue") +
  geom_line(aes(mean_topColi, y), data=new_int, linetype="twodash", color="blue", size=1) + theme_classic() +
  scale_size(limits=c(50,100), breaks=c(60, 80, 100)) + scale_x_continuous(limits=c(0, 0.35)) +
  labs(x="Relative ARG abundance", y="Proportion of resistant clinical isolates", title="A")
```

**Clinical fluoroquinolone resistance**
```{r}
tmp <- ResData %>% filter(`FQ_(n)`>100)
tmp$y = tmp$`FQ_(R%)`/100
breg <- betareg(y ~ mean_topColi, data=tmp, link="loglog")
summary(breg)


new_int <- data.frame(mean_topColi=seq(min(tmp$mean_topColi), max(tmp$mean_topColi)+0.05, 0.05))
new_int$y <- predict(breg, newdata=new_int)

p2 <- ggplot(tmp, aes(mean_topColi, y)) + geom_point(pch=21, size=5, fill="skyblue") +
  geom_line(aes(mean_topColi, y), data=new_int, linetype="twodash", color="blue", size=1) + theme_classic() +
  scale_size(limits=c(50,100), breaks=c(60, 80, 100)) + scale_x_continuous(limits=c(0, 0.35)) +
  labs(x="Relative ARG abundance", y="Proportion of resistant clinical isolates", title="B")
```

**Clinical 3rd generation cephalosporin resistance**
```{r}
tmp <- ResData %>% filter(`3GC_(n)`>100)
tmp$y = tmp$`3GC_(R%)`/100
breg <- betareg(y ~ mean_topColi, data=tmp, link="loglog")
summary(breg)


new_int <- data.frame(mean_topColi=seq(min(tmp$mean_topColi), max(tmp$mean_topColi)+0.05, 0.05))
new_int$y <- predict(breg, newdata=new_int)

p3 <- ggplot(tmp, aes(mean_topColi, y)) + geom_point(pch=21, size=5, fill="skyblue") +
  geom_line(aes(mean_topColi, y), data=new_int, linetype="twodash", color="blue", size=1) + theme_classic() +
  scale_size(limits=c(50,100), breaks=c(60, 80, 100)) + scale_x_continuous(limits=c(0, 0.35)) +
  labs(x="Relative ARG abundance", y="Proportion of resistant clinical isolates", title="C")

```

**Clinical aminoglycoside resistance**
```{r}
tmp <- ResData %>% filter(`AG_(n)`>100)
tmp$y = tmp$`AG_(R%)`/100
breg <- betareg(y ~ mean_topColi, data=tmp, link="loglog")
summary(breg)


new_int <- data.frame(mean_topColi=seq(min(tmp$mean_topColi), max(tmp$mean_topColi)+0.05, 0.05))
new_int$y <- predict(breg, newdata=new_int)

p4 <- ggplot(tmp, aes(mean_topColi, y)) + geom_point(pch=21, size=5, fill="skyblue") +
  geom_line(aes(mean_topColi, y), data=new_int, linetype="twodash", color="blue", size=1) + theme_classic() +
  scale_size(limits=c(50,100), breaks=c(60, 80, 100)) + scale_x_continuous(limits=c(0, 0.35)) +
  labs(x="Relative ARG abundance", y="Proportion of resistant clinical isolates", title="D")
```

**Fig. 1: E. coli clinical resistance models based on ten most common ARGs in E. coli**
```{r, fig.width=10, fig.height=6}
(p1 + p2) /  (p3 + p4)
```


### _E. coli_ clinical resistance models based on intI1 integrase gene 

Next spet was to use _intI1_ integrase gene as a proxy for antibiotic resistance gene prevalence in sewage samples. 
The results from this approach ands Figure 2 from the article below. 

**Clinical aminopenicillin resistance**
```{r}
tmp <- ResData %>% filter(`AP_(n)`>100)
tmp$y = tmp$`AP_(R%)`/100
breg <- betareg(y ~ mean_int, data=tmp, link="loglog")
summary(breg)


new_int <- data.frame(mean_int=seq(min(tmp$mean_int), max(tmp$mean_int)+0.05, 0.05))
new_int$y <- predict(breg, newdata=new_int)

p1 <- ggplot(tmp, aes(mean_int, y)) + geom_point(pch=21, size=5, fill="skyblue") +
  geom_line(aes(mean_int, y), data=new_int, linetype="twodash", color="blue", size=1) + theme_classic() +
  scale_size(limits=c(50,100), breaks=c(60, 80, 100)) + scale_x_continuous(limits=c(0, 0.25)) +
  labs(x=expression(paste("Relative ", italic("intI1"), " abundance", sep=" ")), y="Proportion of resistant clinical isolates", title="A")
```

**Clinical fluoroquinolone resistance**
```{r}
tmp <- ResData %>% filter(`FQ_(n)`>100)
tmp$y = tmp$`FQ_(R%)`/100
breg <- betareg(y ~ mean_int, data=tmp, link="loglog")
summary(breg)


new_int <- data.frame(mean_int=seq(min(tmp$mean_int), max(tmp$mean_int)+0.05, 0.05))
new_int$y <- predict(breg, newdata=new_int)

p2 <- ggplot(tmp, aes(mean_int, y)) + geom_point(pch=21, size=5, fill="skyblue") +
  geom_line(aes(mean_int, y), data=new_int, linetype="twodash", color="blue", size=1) + theme_classic() +
  scale_size(limits=c(50,100), breaks=c(60, 80, 100)) + scale_x_continuous(limits=c(0, 0.25)) +
  labs(x=expression(paste("Relative ", italic("intI1"), " abundance", sep=" ")), y="Proportion of resistant clinical isolates", title="B")
```

**Clinical 3rd generation cephalosporin resistance**
```{r}
tmp <- ResData %>% filter(`3GC_(n)`>100)
tmp$y = tmp$`3GC_(R%)`/100
breg <- betareg(y ~ mean_int, data=tmp, link="loglog")
summary(breg)


new_int <- data.frame(mean_int=seq(min(tmp$mean_int), max(tmp$mean_int)+0.05, 0.05))
new_int$y <- predict(breg, newdata=new_int)

p3 <- ggplot(tmp, aes(mean_int, y)) + geom_point(pch=21, size=5, fill="skyblue") +
  geom_line(aes(mean_int, y), data=new_int, linetype="twodash", color="blue", size=1) + theme_classic() +
  scale_size(limits=c(50,100), breaks=c(60, 80, 100)) + scale_x_continuous(limits=c(0, 0.25)) +
  labs(x=expression(paste("Relative ", italic("intI1"), " abundance", sep=" ")), y="Proportion of resistant clinical isolates", title="C")
```

**Clinical aminoglycoside resistance**
```{r}
tmp <- ResData %>% filter(`AG_(n)`>100)
tmp$y = tmp$`AG_(R%)`/100
breg <- betareg(y ~ mean_int, data=tmp, link="loglog")
summary(breg)


new_int <- data.frame(mean_int=seq(min(tmp$mean_int), max(tmp$mean_int)+0.05, 0.05))
new_int$y <- predict(breg, newdata=new_int)

p4 <- ggplot(tmp, aes(mean_int, y)) + geom_point(pch=21, size=5, fill="skyblue") +
  geom_line(aes(mean_int, y), data=new_int, linetype="twodash", color="blue", size=1) + theme_classic() +
  scale_size(limits=c(50,100), breaks=c(60, 80, 100)) + scale_x_continuous(limits=c(0, 0.25)) +
  labs(x=expression(paste("Relative ", italic("intI1"), " abundance", sep=" ")), y="Proportion of resistant clinical isolates", title="D")
```

**Fig. 2: E. coli clinical resistance models based on intI1 integrase gene**
```{r, fig.width=10, fig.height=6}
(p1 + p2) /  (p3 + p4)
```

### Aggregated resistance model based on intI1 integrase gene

Although the two approaches performed equally well, the _intI1_ approach was chosen due to its simplicity compared to the top ARGs in _E.  coli_. 
So _intI1_ integrase gene abundance was used to model the aggregated resistance index. Results from beta regression model and Figure 3 below. With all data and with Nigeria and Peru removed from the analysis. 

**With all data**
```{r}
tmp <- ResData
tmp$y = tmp$AverageAll/100
breg <- betareg(y ~ mean_int, data=tmp, link="loglog")
summary(breg)


new_int <- data.frame(mean_int=seq(min(tmp$mean_int), max(tmp$mean_int)+0.05, 0.05))
new_int$y <- predict(breg, newdata=new_int)

p1 <- ggplot(tmp, aes(mean_int, y)) + geom_point(pch=21, size=5, fill="skyblue") +
  geom_line(aes(mean_int, y), data=new_int, linetype="twodash", color="blue", size=1) + theme_classic() +
  scale_y_continuous(limits=c(0,0.7)) +
  labs(x=expression(paste("Relative ", italic("intI1"), " abundance", sep=" ")), y="Aggregated proportion of resistant isolates",
       title="A")
```

**With outliers removed**
```{r}
tmp <- ResData
tmp$y = tmp$AverageAll/100
tmp <- tmp %>% filter(!(Country=="Nigeria" | Country=="Peru"))
breg <- betareg(y ~ mean_int, data=tmp, link="loglog")
summary(breg)

new_int <- data.frame(mean_int=seq(min(tmp$mean_int), max(tmp$mean_int)+0.05, 0.05))
new_int$y <- predict(breg, newdata=new_int)

p2 <- ggplot(tmp, aes(mean_int, y)) + geom_point(pch=21, size=5, fill="skyblue") +
  geom_line(aes(mean_int, y), data=new_int, linetype="twodash", color="blue", size=1) + theme_classic() +
  scale_x_continuous(limits=c(0,0.3)) + scale_y_continuous(limits=c(0,0.7)) +
  labs(x=expression(paste("Relative ", italic("intI1"), " abundance", sep=" ")), y="Aggregated proportion of resistant isolates",
       title="B")
```

**Fig. 3: Aggregated resistance model based on intI1 integrase gene.**
```{r, fig.width=10, fig.height=3}
p1 + p2
```

### E. coli clinical resistance models based on combined sewage and socioeconomical data

It is clear from the results that sewage data alone is not enough for modeling clinical resistance prevalence. To improve the performance, we included socioeconomic factors to the models. 
We used GDP, proportion of urban population and basic sanotation index in combination with the sewage data (_intI1_) to model clinical resistance prevalence in _E. coli_ and more broadly using the aggregated resistance index. 

**Amniopenicillin resistance**
```{r}
tmp <- ResData %>% filter(`AP_(n)`>100)
tmp$y = tmp$`AP_(R%)`/100

breg <- betareg(y ~  log(GDP) + urban_population + basic_sanitation + mean_int, data=tmp, link="loglog")
summary(breg)

new_int0 <- data.frame(mean_int=seq(min(tmp$mean_int), max(tmp$mean_int)+0.05, 0.05),
                       urban_population=quantile(tmp$urban_population)[2], basic_sanitation=quantile(tmp$basic_sanitation)[2],
                       GDP=quantile(tmp$GDP)[2])
new_int5 <- data.frame(mean_int=seq(min(tmp$mean_int), max(tmp$mean_int)+0.05, 0.05),
                      urban_population=quantile(tmp$urban_population)[3], basic_sanitation=quantile(tmp$basic_sanitation)[3],
                      GDP=quantile(tmp$GDP)[3])
new_int9 <- data.frame(mean_int=seq(min(tmp$mean_int), max(tmp$mean_int)+0.05, 0.05),
                       urban_population=quantile(tmp$urban_population)[4], basic_sanitation=quantile(tmp$basic_sanitation)[4],
                       GDP=quantile(tmp$GDP)[4])

new_int0$y <- predict(breg, newdata=new_int0)
new_int5$y <- predict(breg, newdata=new_int5)
new_int9$y <- predict(breg, newdata=new_int9)

p1 <- ggplot(tmp, aes(mean_int, y, size=basic_sanitation, fill=urban_population)) + geom_point(pch=21) +
  geom_line(aes(mean_int, y), data=new_int0, linetype="twodash", color="darkblue", size=1) +
  geom_line(aes(mean_int, y), data=new_int5, linetype="twodash", color="blue", size=1) +
  geom_line(aes(mean_int, y), data=new_int9, linetype="twodash", color="lightblue", size=1) + theme_classic() +
  scale_size(limits=c(min(tmp$basic_sanitation),max(tmp$basic_sanitation)),
             breaks=c(60, 80, 100), name="Basic sanitation") +
  scale_x_continuous(limits=c(0, 0.25)) +
  scale_fill_gradient(limits=c(min(tmp$urban_population), max(tmp$urban_population)),
                      breaks=c(20, 40, 60, 80,100), name="Urban population") +
  labs(x=expression(paste("Relative ", italic("intI1"), " abundance", sep=" ")), y="Proportion of resistant clinical isolates",
       title="A")
```

**Fluoroquinolone resistance**
```{r}
tmp <- ResData %>% filter(`FQ_(n)`>100)
tmp$y = tmp$`FQ_(R%)`/100

breg <- betareg(y ~ log(GDP) + urban_population + basic_sanitation + mean_int, data=tmp, link="loglog")
summary(breg)

new_int0 <- data.frame(mean_int=seq(min(tmp$mean_int), max(tmp$mean_int)+0.05, 0.05),
                       urban_population=quantile(tmp$urban_population)[2], basic_sanitation=quantile(tmp$basic_sanitation)[2],
                       GDP=quantile(tmp$GDP)[2])
new_int5 <- data.frame(mean_int=seq(min(tmp$mean_int), max(tmp$mean_int)+0.05, 0.05),
                      urban_population=quantile(tmp$urban_population)[3], basic_sanitation=quantile(tmp$basic_sanitation)[3],
                      GDP=quantile(tmp$GDP)[3])
new_int9 <- data.frame(mean_int=seq(min(tmp$mean_int), max(tmp$mean_int)+0.05, 0.05),
                       urban_population=quantile(tmp$urban_population)[4], basic_sanitation=quantile(tmp$basic_sanitation)[4],
                       GDP=quantile(tmp$GDP)[4])

new_int0$y <- predict(breg, newdata=new_int0)
new_int5$y <- predict(breg, newdata=new_int5)
new_int9$y <- predict(breg, newdata=new_int9)

p2 <- ggplot(tmp, aes(mean_int, y, fill=urban_population, size=basic_sanitation)) + geom_point(pch=21) +
  geom_line(aes(mean_int, y), data=new_int0, linetype="twodash", color="darkblue", size=1) +
  geom_line(aes(mean_int, y), data=new_int5, linetype="twodash", color="blue", size=1) +
  geom_line(aes(mean_int, y), data=new_int9, linetype="twodash", color="lightblue", size=1) + theme_classic() +
  scale_size(limits=c(min(tmp$basic_sanitation),max(tmp$basic_sanitation)),
             breaks=c(60, 80, 100), name="Basic sanitation") +
  scale_x_continuous(limits=c(0, 0.25)) +
  scale_fill_gradient(limits=c(min(tmp$urban_population), max(tmp$urban_population)),
                      breaks=c(20, 40, 60, 80,100), name="Urban population") +
  labs(x=expression(paste("Relative ", italic("intI1"), " abundance", sep=" ")), y="Proportion of resistant clinical isolates",
       title="B")
```

**3rd generation cephalosporins**
```{r}
tmp <- ResData %>% filter(`3GC_(n)`>100)
tmp$y = tmp$`3GC_(R%)`/100

breg <- betareg(y ~  log(GDP) + urban_population + basic_sanitation + mean_int, data=tmp, link="loglog")
summary(breg)

new_int0 <- data.frame(mean_int=seq(min(tmp$mean_int), max(tmp$mean_int)+0.05, 0.05),
                       urban_population=quantile(tmp$urban_population)[2], basic_sanitation=quantile(tmp$basic_sanitation)[2],
                       GDP=quantile(tmp$GDP)[2])
new_int5 <- data.frame(mean_int=seq(min(tmp$mean_int), max(tmp$mean_int)+0.05, 0.05),
                      urban_population=quantile(tmp$urban_population)[3], basic_sanitation=quantile(tmp$basic_sanitation)[3],
                      GDP=quantile(tmp$GDP)[3])
new_int9 <- data.frame(mean_int=seq(min(tmp$mean_int), max(tmp$mean_int)+0.05, 0.05),
                       urban_population=quantile(tmp$urban_population)[4], basic_sanitation=quantile(tmp$basic_sanitation)[4],
                       GDP=quantile(tmp$GDP)[4])

new_int0$y <- predict(breg, newdata=new_int0)
new_int5$y <- predict(breg, newdata=new_int5)
new_int9$y <- predict(breg, newdata=new_int9)

p3 <- ggplot(tmp, aes(mean_int, y, size=basic_sanitation, fill=urban_population)) + geom_point(pch=21) +
  geom_line(aes(mean_int, y), data=new_int0, linetype="twodash", color="darkblue", size=1) +
  geom_line(aes(mean_int, y), data=new_int5, linetype="twodash", color="blue", size=1) +
  geom_line(aes(mean_int, y), data=new_int9, linetype="twodash", color="lightblue", size=1) + theme_classic() +
  scale_size(limits=c(min(tmp$basic_sanitation),max(tmp$basic_sanitation)),
             breaks=c(60, 80, 100), name="Basic sanitation") +
  scale_x_continuous(limits=c(0, 0.25)) +
  scale_fill_gradient(limits=c(min(tmp$urban_population), max(tmp$urban_population)),
                      breaks=c(20, 40, 60, 80,100), name="Urban population") +
  labs(x=expression(paste("Relative ", italic("intI1"), " abundance", sep=" ")), y="Proportion of resistant clinical isolates",
       title="C")
```

**Amoinoglycoside resistance**
```{r}
tmp <- ResData %>% filter(`AG_(n)`>100)
tmp$y = tmp$`AG_(R%)`/100

breg <- betareg(y ~  log(GDP) + urban_population + basic_sanitation + mean_int, data=tmp, link="loglog")
summary(breg)

new_int0 <- data.frame(mean_int=seq(min(tmp$mean_int), max(tmp$mean_int)+0.05, 0.05),
                       urban_population=quantile(tmp$urban_population)[2], basic_sanitation=quantile(tmp$basic_sanitation)[2],
                       GDP=quantile(tmp$GDP)[2])
new_int5 <- data.frame(mean_int=seq(min(tmp$mean_int), max(tmp$mean_int)+0.05, 0.05),
                      urban_population=quantile(tmp$urban_population)[3], basic_sanitation=quantile(tmp$basic_sanitation)[3],
                      GDP=quantile(tmp$GDP)[3])
new_int9 <- data.frame(mean_int=seq(min(tmp$mean_int), max(tmp$mean_int)+0.05, 0.05),
                       urban_population=quantile(tmp$urban_population)[4], basic_sanitation=quantile(tmp$basic_sanitation)[4],
                       GDP=quantile(tmp$GDP)[4])

new_int0$y <- predict(breg, newdata=new_int0)
new_int5$y <- predict(breg, newdata=new_int5)
new_int9$y <- predict(breg, newdata=new_int9)

p4 <- ggplot(tmp, aes(mean_int, y, size=basic_sanitation, fill=urban_population)) + geom_point(pch=21) +
  geom_line(aes(mean_int, y), data=new_int0, linetype="twodash", color="darkblue", size=1) +
  geom_line(aes(mean_int, y), data=new_int5, linetype="twodash", color="blue", size=1) +
  geom_line(aes(mean_int, y), data=new_int9, linetype="twodash", color="lightblue", size=1) + theme_classic() +
  scale_size(limits=c(min(tmp$basic_sanitation),max(tmp$basic_sanitation)),
             breaks=c(60, 80, 100), name="Basic sanitation") +
  scale_x_continuous(limits=c(0, 0.25)) +
  scale_fill_gradient(limits=c(min(tmp$urban_population), max(tmp$urban_population)),
                      breaks=c(20, 40, 60, 80,100), name="Urban population") +
  labs(x=expression(paste("Relative ", italic("intI1"), " abundance", sep=" ")), y="Proportion of resistant clinical isolates",
       title="D")
```

**Fig. 4: E. coli clinical resistance models based on combined sewage and socioeconomical data**
```{r, fig.width=10, fig.height=7}
(p1 + p2) /  (p3 + p4) +
  plot_layout(guides = 'collect') & theme(legend.position='bottom')
```

**Aggregated resistance index**
```{r}
tmp <- ResData
tmp$y = tmp$AverageAll/100
tmp <- tmp %>% filter(!(Country=="Nigeria" | Country=="Peru" | Country=="Kosovo"))

breg <- betareg(y ~  log(GDP) + urban_population + basic_sanitation + mean_int, data=tmp, link="loglog")
summary(breg)

new_int0 <- data.frame(mean_int=seq(min(tmp$mean_int), max(tmp$mean_int)+0.05, 0.05),
                       basic_sanitation=quantile(tmp$basic_sanitation)[2],
                       urban_population=quantile(tmp$urban_population)[2], GDP=quantile(tmp$GDP)[2])
new_int5 <- data.frame(mean_int=seq(min(tmp$mean_int), max(tmp$mean_int)+0.05, 0.05),
                       basic_sanitation=quantile(tmp$basic_sanitation)[3],
                      urban_population=quantile(tmp$urban_population)[3], GDP=quantile(tmp$GDP)[3])
new_int9 <- data.frame(mean_int=seq(min(tmp$mean_int), max(tmp$mean_int)+0.05, 0.05),
                       basic_sanitation=quantile(tmp$basic_sanitation)[4],
                       urban_population=quantile(tmp$urban_population)[4], GDP=quantile(tmp$GDP)[4])

new_int0$y <- predict(breg, newdata=new_int0)
new_int5$y <- predict(breg, newdata=new_int5)
new_int9$y <- predict(breg, newdata=new_int9)
```

**Fig. 5: Aggregated resistance model based on combined sewage and socioeconomical data**
```{r, fig.width=10, fig.height=4}
ggplot(tmp, aes(mean_int, y, size=basic_sanitation, fill=urban_population)) + geom_point(pch=21) +
  geom_line(aes(mean_int, y), data=new_int0, linetype="twodash", color="darkblue", size=1) +
  geom_line(aes(mean_int, y), data=new_int5, linetype="twodash", color="blue", size=1) +
  geom_line(aes(mean_int, y), data=new_int9, linetype="twodash", color="lightblue", size=1) + theme_classic() +
  scale_size(limits=c(min(tmp$basic_sanitation),max(tmp$basic_sanitation)),
             breaks=c(60, 80, 100), name="Basic sanitation") +
  scale_x_continuous(limits=c(0, 0.25)) +
  scale_fill_gradient(limits=c(min(tmp$urban_population), max(tmp$urban_population)),
                      breaks=c(20, 40, 60, 80,100), name="Urban population") +
  labs(x=expression(paste("Relative ", italic("intI1"), " abundance", sep=" ")), y="Aggregated proportion of resistant isolates",
       title="")
```

## Supplementary Materials

### Suppl. Fig 1. 